### ISSUE TITLE
Expand / Contract

### ISSUE THREAD

**Issue by Hecatron** [xif-fr/dokuwiki-plugin-catlist#3]
Hi,
as a feature request, would it be possible to have an expand / contract as an option on folders
similar to the way the indexmenu plugin works?

**Comment by xif-fr** [comment]
Yes, it is possible, and there was such a feature a long time ago, which I removed because it was too bugged. I'll rewrite it in the future.
